Tensile/Tensile.py

```diff
@@ -131,7 +131,8 @@ def splitExtraParameters(par):
     argParser.add_argument("--client-build-path", default=None)
     argParser.add_argument("--client-lock", default=None)
     argParser.add_argument("--prebuilt-client", default=None)
-
+    argParser.add_argument("--asm-cache", dest="AsmCacheFile", action="store", type=str, \
+        help="Path to ASM cache YAML file. If it does not exist, generate the cache. If it does exist, use the cache file")
```

Comment:
Does making this user configurable add unnecessary complexity? Should the ASM cache simply always be written to a specific file in the build directory? If not, what benefit/use case does this offer?

Reply:
I went about it that way initially. For me it was a question of where do we want this file? Do we put it in build_tmp? What if we delete build_tmp? It occurred to me that by the option this way the user can decide where they want the file, its name etc.